addons/website/static/src/scss/website.scss

```diff
@@ -406,6 +406,16 @@ $-header-nav-link-height: $nav-link-height;
     }
 }
 
+@for $index from 1 through 5 {
```

Comment:
@bso-odoo Do we require this SCSS code here? I noticed a similar css [here](https://github.com/odoo/odoo/commit/a48a30f954afcb6ff3a59c4f32b05fd0c2cfcd2b#diff-73090a6cb3708fdfbcfda0141b65c0ad7017af7ba94883e8c3f1d750677df5d7R2053). However, it seems that even without adding this SCSS, everything works correctly I guess.

Reply:
@visp-odoo Yes it is required: without it, if you set a gradient background color on one of the 5 presets, it does not get applied.
So, you need the loop - but you could call the `o-add-gradient` mixin inside the loop instead of the current content.

Reply:
Hello @bso-odoo,
Done changes :+1: 
Thanks in Advance :blush:

Reply:
@visp-odoo Your include applies to... nothing >_<.. if you select a background gradient preset it even fails to generate the CSS.
For each index in the loop, we want to apply it to `.o_cc1`, then `.o_cc2`... etc.
I.e. you should keep the `.o_cc#{$index}` from the previous version.

Reply:
@bso-odoo Done changes :+1: